apps/src/code-studio/initApp/loadApp.js

```diff
@@ -402,78 +402,69 @@ function loadAppAsync(appOptions) {
 
 window.dashboard = window.dashboard || {};
 
-window.apps = {
-
-  // Set up projects, skipping blockly-specific steps. Designed for use
-  // by levels of type "external".
-  setupProjectsExternal: function () {
-    if (!window.dashboard) {
-      throw new Error('Assume existence of window.dashboard');
-    }
```

Comment:
Seems fine to delete this, since it doesn't really do anything. However you'll need to inline it in pixelation.js since `window.apps.setupProjectsExternal` gets called from there also.

Reply:
never mind, it looks like you fixed pixelation.js already.